swagger/messaging/activity_message.json

```diff
@@ -0,0 +1,249 @@
+      "Bearer": {
+        "type": "http",
+        "scheme": "bearer"
+      }
+    },
+    "schemas": {
+      "Response": {
+        "type": "object",
+        "properties": {
+          "success": {
+            "type": "boolean"
+          },
+          "data": {
+            "type": "object",
+            "properties": {
+              "activityMessage": {
+                "$ref": "https://vcita.github.io/developers-hub/entities/messaging/activityMessage.json"
+              }
+            }
+          }
+        },
+        "required": [
+          "success",
+          "data"
+        ]
+      },
+      "ErrorResponse": {
+        "type": "object",
+        "properties": {
+          "success": {
+            "type": "boolean"
+          },
+          "errors": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "code": {
+                  "type": "string"
+                },
+                "message": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "required": [
+          "success",
+          "error"
+        ]
+      },
+      "RequestDto": {
+        "type": "object",
+        "properties": {
+          "business_uid": {
+            "type": "string",
+            "description": "The business that is sending the message."
+          },
+          "staff_uid": {
+            "type": "string",
+            "description": "The staff that is sending the message."
+          },
+          "client_uid": {
+            "type": "string",
+            "description": "Client to send the message to."
+          },
+          "message_uid": {
+            "type": "string",
+            "description": "The entity's unique identifier."
+          },
+          "activity_type": {
+            "type": "string",
+            "enum": [
+              "invite"
+            ],
+            "description": "Activity category type from a closed list."
+          },
+          "activity_action": {
+            "type": "string",
+            "enum": [
+              "schedule"
+            ],
+            "description": "Action related to the activity type property."
+          },
+          "channels": {
+            "type": "object",
+            "description": "Determines in which communication channel to send the message"
+          },
+          "message_text": {
+            "type": "object",
+            "properties": {
+              "body": {
+                "type": "string",
+                "description": "The main content of the message."
+              },
+              "subject": {
+                "type": "string",
+                "description": "A concise summary of the message content."
+              }
+            }
+          },
+          "cta_button_text": {
+            "type": "string",
+            "description": "The text for the Email action button"
```

Comment:
This has no meaning in SMS right? So lets mention it:
In the Email, this text will appear on the button embedded in the email message